lib/expected.ts

```diff
@@ -0,0 +1,74 @@
+  'node-v10.24.1-linuxstatic-arm64':
+    '01bc2cfbf7a7dd1a74201ae34a6cfafc1ad1c6d77039f587111738a81637bc5b',
+  'node-v10.24.1-linuxstatic-x64':
+    'a7bbd62b712b3a7ac54953b646f0802e84bc7ecadb0b8a0756323fcffe3310a5',
+  'node-v10.24.1-macos-x64':
+    'f1e3f50c55890ff238809d58c1b3a30b045e844770120fd69c867ea58d3fb0ec',
+  'node-v10.24.1-win-x64':
+    '958647af177a9089bb4f3495e352d5348a1b42858d0111004ca26c3a2ece3f73',
+  'node-v12.22.1-alpine-arm64':
+    'd4951df9ae437086cbb80c934249a24daaaaa71f67845a07b0c0a0af3cc48d56',
+  'node-v12.22.1-alpine-x64':
+    '19d2baad1d8b75221239c1fe5c2467b4ddb07fc4b02e0a034cedcb28f0d6e400',
+  'node-v12.22.1-linux-arm64':
+    'e642dcecaf6052aeca62943aa54c28464571232b753a27cb5698341760bf4234',
+  'node-v12.22.1-linux-x64':
+    '20911af9c855edc3e1e0f33032fcedc59487fc3d908ca78fc75a6d81d88ee8c7',
+  'node-v12.22.1-linuxstatic-arm64':
+    '2265d1d44a6715d10621cdbcf6880b1f3a7173950c027be9014b9eabd75ad6e4',
+  'node-v12.22.1-linuxstatic-x64':
+    '4fbec04273f3b3c2467baaaeec6cdca48cbc57fed07627872345583191dcbb11',
+  'node-v12.22.1-macos-x64':
+    'c7d11d7c427967778c2621870e45ac98b98350f44e992f152de55bf919a737e6',
+  'node-v12.22.1-win-x64':
+    '48637669a8418e33edbef50f7331e337dfd644783c9257a386dac846610c613c',
+  'node-v14.16.1-alpine-arm64':
+    'c074af7d4f4d5c520df9135bc556b2447843c5a469175a7c479f16d58cc073d2',
+  'node-v14.16.1-alpine-x64':
+    '0ea2a4f5edf3b9f4f3350fbb2db7d5f65cf6f433aa0da986cc1e5d99e422b78a',
+  'node-v14.16.1-linux-arm64':
+    'b459de37a899ced574e97a3d2977da6f91559dd3a0f6ecce59610ecafecfaf5b',
+  'node-v14.16.1-linux-x64':
+    '77ad9b663d6f4b5d8c327da2e25e2be5ec6749cc4107635ba854a0d3d3f489fd',
+  'node-v14.16.1-linuxstatic-arm64':
+    '3768d90c502883bf61041b7cbade37021d18c651b50ee597445311991c122e06',
+  'node-v14.16.1-linuxstatic-x64':
+    'a64ffe8250af74ff7bfd352d469a3c41930de6e13efdff37ca1b7524ab335053',
+  'node-v14.16.1-macos-x64':
+    'b13dfc538121a8e19ae6bd6201bdbeca88b6dfd7cb37f5835d8c9e28f9a662c2',
+  'node-v14.16.1-win-arm64':
+    'dac41cc2f5ffa90900939b325464e316c38c3b6514f6285c9bd2701c3c774a9a',
+  'node-v14.16.1-win-x64':
+    '67508e6c5d808887fdb623f21c52add1de531ebbb690971f6780c60dc32c3b8f',
+  'node-v8.17.0-alpine-arm64':
+    '807df81524ec8e1e266ac7fbed434c6b2281ae20b9fa7eaa524de90f3330c4d7',
+  'node-v8.17.0-alpine-x64':
+    '83a3914de57ee6be7d68ebaac8a10c1a2972d554800f1bee218cc4a23650e0fb',
+  'node-v8.17.0-linux-arm64':
+    'f06855896bfa10bead1f08fac080305fb6fbfb2cc691168a3f0f0e834e12bfba',
+  'node-v8.17.0-linux-x64':
+    '14d75d43de1ff86469d354bf42a83b9494e09502fa7bc23a975e2cb82b1608b0',
+  'node-v8.17.0-linuxstatic-arm64':
+    '84de8fe30b2bd1dcb3615cf1d1b538aa48e1fcf66620ef97dce6b7ae85b45025',
+  'node-v8.17.0-linuxstatic-x64':
+    '5206878079f160e75a02ad33b7559b4a869e8181ee03d51d7211b52995f9ca7b',
+  'node-v8.17.0-macos-x64':
+    'dffa71e39100f4daa57de73fda7b4debecd09f552b15cf11854c8475380d3817',
+  'node-v8.17.0-win-x64':
```

Comment:
Should we auto-generate these so it doesn't require a PR for every release? Or not doable?

Reply:
The whole point is to hardcode the hashes in the source code. So we do have to make a commit. 

You can get the hashes simply by `cat *.sha256sum` after downloading from Github Actions. Then, it is simple RegEx (to convert to object). 

To ease the process, I can make the CI print out the hashes in the log.